launcher/main.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * main.h, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+#pragma once
+
+#include "StdInc.h"
```

Comment:
Again is it ok to include precompiled header to another header?

Reply:
it's ok but can be removed, thanks

Reply:
hm, looking at https://github.com/vcmi/vcmi/pull/776/commits/38ecb2a699aa72c3033e51c3ed1f1adc0bf05ce9 I'm not sure that it's a good idea...